.ignore

```diff
@@ -4,3 +4,5 @@ migrations
 
 *.xml
 *.json
+*.po
+*.mo
```

Comment:
does this need to go in here?

Reply:
Yes, I should have put it in a seperate commit, but it makes sense to not search the po files.